libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -311,6 +308,26 @@ const decorateMenu = (config) => logErrorFor(async () => {
     // Content has been fetched dynamically, need to decorate links
     decorateLinks(menuTemplate);
     await decorateColumns({ content: menuContent });
+
+    if (getActiveLink(menuTemplate) instanceof HTMLElement) {
+      // Special handling on desktop, as content is loaded async;
+      // bolding the item text would normally push the content
+      // to the right, potentially causing CLS
+      const resetActiveState = () => {
+        config.template.style.width = '';
+        config.template.classList.remove(selectors.deferredActiveNavItem.slice(1));
+      };
+
+      if (isDesktop.matches) {
```

Comment:
Noticed your message 
> It gets removed once the viewport switches to mobile or the content of the navigation is overflowing

but I think we should keep the bold item in mobile to be consistent with existing implementation.